docs/en-US/component/tour.md

```diff
@@ -57,6 +57,16 @@ tour/indicator
 
 :::
 
+## Target
+
+Various parameter passing types of target. The string and Function types are supported since ^(2.5.2).
```

Comment:
The version tag here is placed after the `## Target` to be more obvious.

Reply:
Is that so?
```
## Target ^(2.5.2)

Various parameter passing types of target. The string and Function types are supported since ^(2.5.2).
```
Do we still retain the sentence `The string and Function types are supported since ^(2.5.2)`?

Reply:
Let's keep it like this for the time being. `target` existed before 2.5.2, so adding tags directly doesn't seem right.

Reply:
ok